packages/admin-next/dashboard/src/i18n/translations/en.json

```diff
@@ -1130,11 +1167,11 @@
       "conditions": {
         "rules": {
           "title": "Who can use this code?",
-          "description": "Is the customer allowed to add the promotion code? Discount code can be used by all customers if left untouched."
+          "description": "Which customer is allowed to use the promotion code? Promotion code can be used by all customers if left untouched."
         },
         "target-rules": {
```

Comment:
not related to your PR, but keys shouldn't use kebab case. But feel free to leave it, and we can clean it up later.